docs/modules/ROOT/pages/how-tos/exoscale/decommission.adoc

```diff
@@ -64,16 +69,44 @@ TF_VAR_lb_exoscale_api_secret
 EOF
 ----
 
+. Set downtimes in https://monitoring.vshn.net[Icinga2] for both LBs
+
 include::partial$setup_terraform_exoscale.adoc[]
 
+. Grab location of LB backups and potential Icinga2 satellite host before decommissioning VMs.
++
+[source,shell]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+vault login -method=ldap username=<your.name>
+declare -a LB_FQDNS
+for id in 0 1; do
+  LB_FQDNS[$id]=$(terraform state show module.cluster.exoscale_domain_record.lb[$id] | grep hostname | cut -d'=' -f2 | tr -d ' "\r\n')
+done
+for lb in ${LB_FQDNS[*]}; do
+  ssh "${lb}" "sudo grep 'server =' /etc/burp/burp.conf && sudo grep 'ParentZone' /etc/icinga2/constants.conf"
+done
+----
+
+. Set downtimes for LBs in https://monitoring.vshn.net[Icinga2].
```

Comment:
Again? Already done at line 72.

Reply:
Once is enough, I removed the step at line 72 in the latest version